core/src/test/java/jenkins/org/apache/commons/validator/routines/DomainValidatorTest.java

```diff
@@ -25,9 +25,7 @@
 import java.io.InputStream;
 import java.lang.reflect.Field;
 import java.lang.reflect.Modifier;
-import java.net.HttpURLConnection;
-import java.net.IDN;
-import java.net.URL;
+import java.net.*;
```

Comment:
That syntax is banned by checkstyle. How did you compile the PR successfully?

Reply:
@NotMyFault I have fixed the issue. Please re-review it. I compiled the PR as per instructions mentioned at [contribution guidelines](https://github.com/abhishekmaity/jenkins/blob/task-1/CONTRIBUTING.md) using openJDK 17 using IntelliJ IDEA

Reply:
> I compiled the PR as per instructions mentioned at [contribution guidelines](https://github.com/abhishekmaity/jenkins/blob/task-1/CONTRIBUTING.md?rgh-link-date=2023-09-22T01%3A04%3A48Z) using openJDK 17 using IntelliJ IDEA

Jenkins does not compile if you invoke verification and there are code style issues.